code/modules/mining/mine_turfs.dm

```diff
@@ -37,30 +37,16 @@
 
 /turf/simulated/mineral/New()
 	..()
+	icon_state = "rock[rand(0,4)]"
 	spawn(0)
 		MineralSpread()
```

Comment:
Can you move this to initialize as well? Having spawns in New() doesnt seem like a good idea